utility/README.md

```diff
@@ -1,679 +1,1309 @@
+#### 3.5.3 Application w/o Gateway
 
+An Application that chooses to operate without a Gateway is responsible for dispatching sessions and signing RPC requests on its own. To do so, it will need to maintain a Pocket Full Node or a Pocket Light Client.
 
-```
-# Application(Edit)StakeMsg Interface
-type ApplicationStakeMsg interface {
-  GetPublicKey()  PublicKey       # identity of edited ServiceNode
-  GetStakeAmount() BigInt         # must be greater than or equal to the current value
-  GetRelayChains() RelayChain[]   # may be modified
-  GetGeoZone() LocationIdentifier # may be modified
-  GetNumOfNodes() int8            # may be modified
-}
-```
+```mermaid
+sequenceDiagram
+    actor AC as Application / Client
+    actor LN as Local Pocket Node / <br>Local Light Client
+    actor SN as Servicer
 
+    AC ->> +LN: StartSession()
+    LN ->> -AC: SessionData([ServicerIDs], ...)
 
-An Application is able to submit an UnstakeMsg to exit the network and remove themself from the network. After a successful UnstakeMsg, the Application is no longer eligible to consume Web3 traffic from ServiceNodes. After ApplicationUnstakingTime elapses, any Stake amount left is returned to the custodial account.
+    loop Session Duration
+        AC ->> AC: Sign Request
 
+        AC ->>+ SN: SignedRequest
+        SN ->> SN: Validate Signature<br>& Session Limits
+        SN ->> SN: Handle Request<br>& Sign Response
+        SN ->>- AC: SignedResponse
 
-```
-# ApplicationUnstake Interface
-type ApplicationUnstake interface {
-  GetAddress()  Address       # The address of the Application unstaking
-}
+        AC ->> AC: Process response
+    end
 ```
 
+#### 3.5.4 Application Delegation
 
-Application Stake burn is a necessary mechanism to ensure an equilibratory economy at network maturity. Given the timeline for network maturity is approximated to be a decade from the time of publication, this document does not detail a non-inflationary economic scenario. In the inflationary phase of Pocket Network, Application Stake burn rate is able to be controlled through the AppBurnPerSession parameter which is expected to be set near zero. In practice, though the burn value is expected to be negligible, Application stake is able to be burned over a function of time. As described above, MaxRelaysPerSession is generated by the StakeAmount of the Application and the Application burn will affect this value as the StakeAmount changes. 
+An Application that chooses to delegate trust to a Gateway will need to submit a one-time `DelegateMsg` transaction to delegate trust from the Application to the Gateway. It must include the PublicKey of the Gateway and be signed by the Application.
```

Comment:
Comment: Missing details on how delegation “state” will be stored on the persistent layer. (Maybe I am just missing something, but the entire persistent layer does not document the schema specifications). Why do we define [account and pool](https://github.com/pokt-network/pocket-network-protocol/tree/issues/23/aat/utility#371-account-structure), but not any other actor protos/structs in this spec?

Reply:
For consistency purposes, I'd remove them besides the `SendTransaction` piece, but otherwise LGTM